src/components/modals/TradePreviewModalGP.vue

```diff
@@ -435,6 +469,55 @@ export default defineComponent({
       emit('close');
     }
 
+    function cofirmPriceUpdate() {
+      priceUpdated.value = false;
+      priceUpdateAccepted.value = true;
+      lastQuote.value = props.trading.getQuote();
+    }
+
+    function handlePriceUpdate() {
+      if (lastQuote.value != null) {
+        const newQuote = props.trading.getQuote();
+
+        if (props.trading.exactIn.value) {
+          priceUpdated.value = bnum(1)
+            .minus(
+              bnum(lastQuote.value.minimumOutAmount).div(
+                newQuote.minimumOutAmount
+              )
+            )
+            .abs()
+            .gt(PRICE_UPDATE_THRESHOLD);
```

Comment:
The price update will show if there is a movement of `PRICE_UPDATE_THRESHOLD` (2% currently) in either direction.